lms/views/helpers/_via.py

```diff
@@ -53,26 +45,17 @@ def __init__(
         }
 
         self.legacy_mode = legacy_mode
-        self.via_rewriter_mode = via_rewriter_mode
 
     def url_for(self, doc):
         if self.legacy_mode:
             return self._legacy_via_url(doc.url)
 
         if doc.is_html:
-            if self.via_rewriter_mode:
-                return self._pywb_style_url(
-                    self.service_url._replace(path="/html/v/").geturl(), doc.url
-                )
-
             return self._legacy_via_url(doc.url)
```

Comment:
This means we don't direct straight to ViaHTML at any time, but this is going to change pretty soon, when we strip out all the legacy Via stuff.